docs/pages/world/modules.mdx

```diff
@@ -5,6 +5,24 @@ This is somewhat similar to one of the use cases for [foundry scripts](https://b
 
 ## Module installation
 
+The easiest way to install modules is to edit [the config file](/config).
+At the top level you specify `modules`, a list of modules to install.
+Every module is a struct with three parameters:
+
+- `artifactPath` - relative path to the module's compiled JSON artifact (usually in `out`) or an import path if using a module from an npm package.
+  This path is resolved using [Node's module `require` API](https://nodejs.org/api/modules.html#modulerequireid).
+- `root` - whether the module is to be installed with [root namespace privileges](/world/systems#root-systems).
+  The default is `false`.
+- `args` - a list of arguments to give the module's install function.
+  Each argument is a structure with two fields:
+  - `type` - the Solidity data type
+  - `value` - the value
+    To encode a complex data type, such as a structure or an array, you can use Viem's [`encodeAbiParameters`](https://viem.sh/docs/abi/encodeAbiParameters.html).
```

Comment:
can we link to the modules section of the config reference to explain this rather than duplicating it here?